src/tests/tests_general.jl

```diff
@@ -202,7 +202,19 @@ function ManifoldTests.test_manifold(
             atol = atolp1p2,
             rtol = rtolp1p2,
         )
-        Test.@test isapprox(M, pts[1], exp(M, pts[2], X2); atol = atolp1p2, rtol = rtolp1p2)
+        if VERSION >= v"1.5" && isa(M, Union{Grassmann,GeneralizedStiefel})
```

Comment:
Since the test suite passed before these fixes, it would be nice to have some tests that would have failed before the fixes and now pass.

Reply:
Actually, the old test suite (1.0 and 1.4) would pass again, it's just nightly (for which this if was already in) and 1.5 (which is now in the test suite) that require this.

Reply:
The above comment is not about this line of code but about tests in general. There was a bug, but the test suite passed. So there are two separate problems. So far from what I can tell this PR fixes the bug but not the inadequacy in the test suite that allowed us to miss the bug.

Reply:
Ah, you're right – I was too stuck to this line. I will add something adapted from your example then.

Reply:
Thanks! I'll go ahead and approve. Once you're happy with that, feel free to merge!